frontend/compositions/visualizations/map/BaseMap.tsx

```diff
@@ -16,101 +12,70 @@ import { FakeData } from "../utilities/chartTypes"
-  const colorGradient = d3.interpolate("#7caed7", "#303463")
-
-  const svg = d3.select(baseMapRef.current)
+  const valueScale = scaleLinear().domain([0, 5]).range([0, 1])
+  const colorGradient = interpolate("#7caed7", "#303463")
 
+  const svg = select(baseMapRef.current)
   useEffect(() => {
-    if (!geoData) return
-
+    // if (!geoData) return
     // Set the title after loading data to facilitate testing
     svg.attr("title", "US Map")
 
     /* Definitions */
-    const defs = svg.append("defs")
-
-    const strokeShape = defs.append("filter").attr("id", "strokeShape")
-
-    strokeShape
-      .append("feMorphology")
-      .attr("operator", "erode")
-      .attr("radius", 1)
-      .attr("result", "erode")
-
-    strokeShape
-      .append("feGaussianBlur")
-      .attr("in", "erode")
-      .attr("stdDeviation", "3")
-      .attr("result", "blurFilter")
-
-    strokeShape
-      .append("feColorMatrix")
-      .attr("id", "colorMatrix2")
-      .attr("in", "blurFilter")
-      .attr("mode", "matrix")
-      .attr("values", "1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 14 -7")
-      .attr("result", "colorMatrix2")
-
-    strokeShape
-      .append("feGaussianBlur")
-      .attr("in", "erode")
-      .attr("stdDeviation", "2")
-      .attr("result", "blurFilter2")
-
-    strokeShape
-      .append("feComposite")
-      .attr("in", "colorMatrix2")
-      .attr("in2", "blurFilter2")
-      .attr("operator", "out")
-      .attr("result", "compositedStroke")
-
-    strokeShape
-      .append("feBlend")
-      .attr("in", "colorMatrix2")
-      .attr("in2", "compositedStroke")
-      .attr("mode", "multiply")
+    // const defs = svg.append("defs")
+
+    // const strokeShape = defs.append("filter").attr("id", "strokeShape")
+
+    // strokeShape
+    //   .append("feMorphology")
+    //   .attr("operator", "erode")
+    //   .attr("radius", 1)
+    //   .attr("result", "erode")
+
+    // strokeShape
+    //   .append("feGaussianBlur")
+    //   .attr("in", "erode")
+    //   .attr("stdDeviation", "3")
+    //   .attr("result", "blurFilter")
+
+    // strokeShape
+    //   .append("feColorMatrix")
+    //   .attr("id", "colorMatrix2")
+    //   .attr("in", "blurFilter")
+    //   .attr("mode", "matrix")
+    //   .attr("values", "1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 14 -7")
+    //   .attr("result", "colorMatrix2")
+
+    // strokeShape
+    //   .append("feGaussianBlur")
+    //   .attr("in", "erode")
+    //   .attr("stdDeviation", "2")
+    //   .attr("result", "blurFilter2")
+
+    // strokeShape
+    //   .append("feComposite")
+    //   .attr("in", "colorMatrix2")
+    //   .attr("in2", "blurFilter2")
+    //   .attr("operator", "out")
+    //   .attr("result", "compositedStroke")
+
+    // strokeShape
+    //   .append("feBlend")
+    //   .attr("in", "colorMatrix2")
+    //   .attr("in2", "compositedStroke")
+    //   .attr("mode", "multiply")
 
     const paths = svg.append("g").attr("id", "paths")
+
     /* SVG Body */
```

Comment:
even here I'd be inclined to remove the comment - I think we could capture what this comment is communicating by naming "paths" a little more informatively!

Reply:
this comment is marking the text flow of the document itself, to help read/scan the text, not describing the function of the code... 'paths' is the right name for that variable, and different text wouldn't solve the problem anyway-- I think the solution would actually be ... a longer/more informative comment ... ??  xD